stdlib/public/Concurrency/Actor.cpp

```diff
@@ -649,17 +649,14 @@ class alignas(sizeof(void *) * 2) ActiveActorStatus {
     PriorityShift = 0x8,
   };
 
-#if SWIFT_CONCURRENCY_ENABLE_PRIORITY_ESCALATION && SWIFT_POINTER_IS_4_BYTES
-  uint32_t Flags;
-  dispatch_lock_t DrainLock;
-  uint32_t Unused;
-#elif SWIFT_CONCURRENCY_ENABLE_PRIORITY_ESCALATION && SWIFT_POINTER_IS_8_BYTES
   uint32_t Flags;
+#if SWIFT_CONCURRENCY_ENABLE_PRIORITY_ESCALATION
   dispatch_lock_t DrainLock;
-#elif !SWIFT_CONCURRENCY_ENABLE_PRIORITY_ESCALATION && SWIFT_POINTER_IS_4_BYTES
-  uint32_t Flags;
-#else /* !SWIFT_CONCURRENCY_ENABLE_PRIORITY_ESCALATION && SWIFT_POINTER_IS_8_BYTES */
-  uint32_t Flags;
+#define ACTIVE_ACTOR_STATUS_HAS_UNUSED (SWIFT_POINTER_IS_4_BYTES)
+#else
+#define ACTIVE_ACTOR_STATUS_HAS_UNUSED (SWIFT_POINTER_IS_8_BYTES)
+#endif
```

Comment:
Please specify in comments what `if` clause each `else` and `endif` maps to. It's hard to do the pairwise matching otherwise

Reply:
Just to make sure I understand what you expect, do you mean like this?
```
#if SWIFT_CONCURRENCY_ENABLE_PRIORITY_ESCALATION
  dispatch_lock_t ExecutionLock;
#define ACTIVE_ACTOR_STATUS_HAS_UNUSED (SWIFT_POINTER_IS_4_BYTES)
#else // !SWIFT_CONCURRENCY_ENABLE_PRIORITY_ESCALATION
#define ACTIVE_ACTOR_STATUS_HAS_UNUSED (SWIFT_POINTER_IS_8_BYTES)
#endif // SWIFT_CONCURRENCY_ENABLE_PRIORITY_ESCALATION
```

Reply:
Yes please.

Reply:
I've reverted this part in favor of initializing in the declaration